src/components/victory-gauge.js

```diff
@@ -42,9 +49,25 @@ export default class VictoryGauge extends React.Component {
   };
 
   static propTypes = {
+    //tickValues array of all values to be input as ticks
```

Comment:
use `/** Description of prop "foo". */` style for comments so that our auto-docs tool can pick it up